hyperlink/_url.py

```diff
@@ -1560,7 +1560,7 @@ def get(self, name):
         """
         return [value for (key, value) in self.query if name == key]
 
-    def remove(self, name):
+    def remove(self, name, value=_UNSET):
```

Comment:
You need to update the Args list and the description for the new parameter, I think?